assets/Product.wxs

```diff
@@ -1,12 +1,10 @@
-﻿<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
-
+﻿<Wix
+  xmlns="http://schemas.microsoft.com/wix/2006/wi">
```

Comment:
Unnecessary change?

Reply:
Might have accidentally hit Enter on that line.  Will fix.